index.bs

```diff
@@ -640,13 +655,6 @@ by using the raw output of a gyroscope
 and waiting for quick rotational movements of the head
 to compensate for drift.
 
-Note: [=sensors=] created through [=sensor fusion=] are sometimes
-called virtual or synthetic sensors.
-However, the specification doesn't make any practical differences between them,
-preferring instead to differentiate [=sensors=] as to whether they describe
```

Comment:
and instead differentiates sensors based on whether

Reply:
this part is now removed